src/libs/antares/study/runtime/runtime.cpp

```diff
@@ -483,7 +352,6 @@ bool StudyRuntimeInfos::loadFromStudy(Study& study)
     logs.info() << "     thermal clusters: " << thermalPlantTotalCount;
     logs.info() << "     thermal clusters (must-run): " << thermalPlantTotalCountMustRun;
     logs.info() << "     short-term storages: " << shortTermStorageCount;
-    logs.info() << "     binding constraints: " << bindingConstraints.size();
```

Comment:
This info is actually quite useful, please keep it if possible

Reply:
Done